### PR TITLE
Use `auditlogs.enabled` as flag to enable auditlogs across services

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Review by merlin-northern — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -138,6 +138,7 @@ The following table lists the global parameters supported by the chart and their
 | Parameter | Description | Default |
 | -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
 | `global.enterprise` | Enable the enterprise features | `true` |
+| `global.auditlogs` | Enabled the auditlogs feature (requires enterprise) | `true` |
```
Comment (merlin-northern) [inline README.md:141]: I wonder gui uses `.Values.auditlogs.enabled`

Comment (merlin-northern) [reply README.md:141]: andso does workflows-worker

Comment (tranchitella) [reply README.md]: You are right, doesn't make sense to have both flags.
I reworked the PR, let's use `auditlogs.enabled` everywhere.

File: mender/templates/gui-deploy.yaml [hunk 2]
```diff
@@ -79,7 +79,7 @@ spec:
         - name: HAVE_MULTITENANT
           value: {{ .Values.global.enterprise | quote }}
         {{- end }}
-        {{- if .Values.auditlogs.enabled }}
+        {{- if and .Values.auditlogs.enabled .Values.global.enterprise }}
         - name: HAVE_AUDITLOGS
           value: {{ .Values.auditlogs.enabled | quote }}
```
Comment (bitcloud) [inline mender/templates/gui-deploy.yaml:84]: This variable doesn't make sense. If it is false then the whole block doesn't exist. so why not do `value: "true"` as in all the other cases?

Comment (tranchitella) [reply mender/templates/gui-deploy.yaml]: @bitcloud you are right, I force-pushed setting the value to `"true"`

**Review by merlin-northern — APPROVED** [review]
looks good to me

**Pull request merged**